libs/portal-framework-core/src/plugins/remoteComponentLoader.tsx

```
@@ -149,20 +143,25 @@ export function createBridgeComponent<T>(
+    {
+      ref,
+      ...props
+    }: T & {
+      ref: React.RefObject<ComponentRef>;
+    }
+  ) => {
+    return store
+      .getRegisteredContextIds()
+      .reduce(
+        (children, contextId) => (
+          <RemoteContextBridge contextId={contextId}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)